### PR TITLE
SAIC-354: Filtering tenant in tests

### PR DESCRIPTION
**Pull request by realsystem**

### CODE REVIEW

**Comment by realsystem** [comment]
ready for review

File: devlab/tests/filtering_utils.py [hunk 1]
```diff
@@ -61,6 +61,20 @@ def filter_images(self, src_data_list):
...
         return [src_data_list, popped_img_list]
 
+    def filter_tenants(self, src_data_list):
+        loaded_data = self.load_file('configs/filter.yaml')
+        filter_dict = loaded_data[0]
+        popped_ten_list = []
+        src_data_list = [x.__dict__ for x in src_data_list]
+        if 'tenants' not in filter_dict:
+            return [src_data_list, []]
+        for ten in src_data_list[:]:
```
Comment (roman-verchikov) [inline devlab/tests/filtering_utils.py]: `for ten in src_data_list:` would work fine too

Comment (roman-verchikov) [inline devlab/tests/filtering_utils.py]: Also `ten` name for variable is misleading, it reads as "number 10", not "tenant", I would suggest renaming to `t`.

**Comment by roman-verchikov** [comment]
+1

**Comment by roman-verchikov** [comment]
can be merged

**Pull request merged**